src/App.js

```diff
@@ -1,4 +1,3 @@
-import React from "react";
 import { BrowserRouter, Route, Routes } from "react-router-dom";
```

Comment:
I kept getting an error when I tried to useLocation() in this file. It turns out we need to remove BroswerRouter from App.js and put it back in index.js. This resolved the error. I believe we were getting an error and that's why we moved it in the first place? In any case, moving it works now, so go ahead and make that change. Let me know if you have any insight into this though!